build.sbt

```diff
@@ -0,0 +1,73 @@
+import sbtrelease.ReleaseStateTransformations._
+
+name:= "content-api-firehose-client"
+organization := "com.gu"
+scalaVersion := "2.11.8"
+scalacOptions ++= Seq("-feature", "-deprecation", "-unchecked", "-target:jvm-1.8", "-Xfatal-warnings")
+scalacOptions in doc in Compile := Nil
+
+pomExtra := (
+<url>https://github.com/guardian/content-api-firehose-client</url>
+  <developers>
+    <developer>
+      <id>LATaylor-guardian</id>
+      <name>Luke Taylor</name>
+      <url>https://github.com/LATaylor-guardian</url>
+    </developer>
+  </developers>
+)
+publishTo <<= version { v =>
```

Comment:
Same goes for `publishMavenStyle := true` and `pomIncludeRepository := { _ => false }`